.travis.yml

```diff
@@ -6,6 +6,19 @@ branches:
 
 language: objective-c
 
+env:
+  matrix:
+    - SDK=iphonesimulator8.1 OS=8.1 NAME=iphone 6
+    - SDK=iphonesimulator8.2 OS=8.2 NAME=iphone 6
+    - SDK=iphonesimulator8.3 OS=8.3 NAME=ihpone 6
+    - SDK=iphonesimulator8.4 OS=8.4 NAME=iphone 6
```

Comment:
This might take too long to run for each pull request. I think ```8.4``` and ```9.0``` should be enough, across ```iPhone``` and ```iPad```.